From 59eab38fd20c3c600eb61d4ea06b86ad65ef29d4 Mon Sep 17 00:00:00 2001
From: Dirk Thomas <dthomas@osrfoundation.org>
Date: Fri, 26 Dec 2014 10:12:50 -0800
Subject: [PATCH] disable CATKIN_BUILD_BINARY_PACKAGE for jade on fedora

---
 rpm/template.spec.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index 6fb5033..2b6d3d9 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -29,7 +29,7 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
         -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
         -DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
         -DSETUPTOOLS_DEB_LAYOUT=OFF \
-        -DCATKIN_BUILD_BINARY_PACKAGE="1" \
+#        -DCATKIN_BUILD_BINARY_PACKAGE="1" \
 
 make %{?_smp_mflags}
 
-- 
1.9.1

