From d9485b864d3e04206b5672c1c904bd8208db3f8f Mon Sep 17 00:00:00 2001
From: Dirk Thomas <dthomas@osrfoundation.org>
Date: Fri, 26 Dec 2014 10:12:50 -0800
Subject: [PATCH] disable CATKIN_BUILD_BINARY_PACKAGE for jade on fedora

---
 rpm/template.spec.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index a65a2a8..f60445f 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -30,7 +30,7 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
         -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
         -DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
         -DSETUPTOOLS_DEB_LAYOUT=OFF \
-        -DCATKIN_BUILD_BINARY_PACKAGE="1" \
+#        -DCATKIN_BUILD_BINARY_PACKAGE="1" \
 
 make %{?_smp_mflags}
 
-- 
2.7.4

