From 3d5849a984c72e824ee11bfb4ade1a6399c83870 Mon Sep 17 00:00:00 2001
From: Dirk Thomas <dthomas@willowgarage.com>
Date: Fri, 22 Mar 2013 14:08:41 -0700
Subject: [PATCH 2/2] fix rule file for groovy

---
 debian/rules.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 8de67c5..fc9408b 100644
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -25,9 +25,9 @@ override_dh_auto_configure:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
-#		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+#		-DCATKIN_BUILD_BINARY_PACKAGE="1"
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
1.7.10.4

